python/equistore-operations/equistore/operations/pow.py

```diff
@@ -1,42 +1,74 @@
-from equistore.core import TensorBlock, TensorMap
+from typing import List, Union
 
 from . import _dispatch
+from ._classes import TensorBlock, TensorMap, check_isinstance, torch_jit_is_scripting
+from ._utils import (
+    _check_blocks_raise,
+    _check_same_gradients_raise,
+    _check_same_keys_raise,
+)
 
 
-def pow(A: TensorMap, B: float) -> TensorMap:
-    r"""Return a new :class:`TensorMap` with the same metadata of ``A``
-    and the values being the element-wise ``B``-power of ``A.values``.
+def divide(A: TensorMap, B: Union[float, int, TensorMap]) -> TensorMap:
```

Comment:
why ``divide`` is in ``pow.py``? maybe copy and paste?

Reply:
Yes 😄